hoomd/mpcd/VirtualParticleFiller.cc

```diff
@@ -77,6 +54,20 @@ void mpcd::VirtualParticleFiller::setType(unsigned int type)
     m_type = type;
     }
 
+unsigned int mpcd::VirtualParticleFiller::computeFirstTag(unsigned int *N_fill)
```

Comment:
This should not be a pointer, just use the value:
```suggestion
unsigned int mpcd::VirtualParticleFiller::computeFirstTag(unsigned int N_fill)
```